src/main/java/pages/BasePage.java

```diff
@@ -18,6 +20,24 @@ public BasePage(WebDriver driver){
     public void waitVisibility(By by){
         wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(by));
     }
+
+    public void clickElement(By elementBy){
+        waitVisibility(elementBy);
+        driver.findElement(elementBy).click();
+    }
+    public void readText(By elementBy){
```

Comment:
ReadText method should return String value as result of method execution.

Reply:
Corrected!